data/filter.go

```diff
@@ -72,75 +72,76 @@ var (
 	// Bulletin - Search information specific for statistical bulletins
 	Bulletin = ContentType{
 		LocaliseKeyName: "StatisticalBulletin",
-		Type:            "bulletin",
-		SubTypes:        []string{"bulletin"},
+		Group:           "bulletin",
+		Types:           []string{"bulletin"},
 	}
 
 	// Article - Search information specific for articles
 	Article = ContentType{
 		LocaliseKeyName: "Article",
-		Type:            "article",
-		SubTypes:        []string{"article", "article_download"},
+		Group:           "article",                               //Type
+		Types:           []string{"article", "article_download"}, //SubType
 	}
 
 	// Compendium - Search information specific for compendium
 	Compendium = ContentType{
 		LocaliseKeyName: "Compendium",
-		Type:            "compendia",
-		SubTypes:        []string{"compendium_landing_page"},
+		Group:           "compendia",
+		Types:           []string{"compendium_landing_page"},
 	}
 
 	// TimeSeries - Search information specific for time series
 	TimeSeries = ContentType{
 		LocaliseKeyName: "TimeSeries",
-		Type:            "time_series",
-		SubTypes:        []string{"timeseries"},
+		Group:           "time_series",
+		Types:           []string{"timeseries"},
 	}
 
 	// Datasets - Search information specific for datasets
 	Datasets = ContentType{
 		LocaliseKeyName: "Datasets",
-		Type:            "datasets",
-		SubTypes:        []string{"dataset_landing_page", "timeseries_dataset"},
+		Group:           "datasets",
+		Types:           []string{"dataset_landing_page", "reference_tables"},
```

Comment:
Im not sure why this has changed back to `reference_tables` from `timeseries_dataset`

Reply:
No idea either but this has been updated